src/NServiceBus.Metrics/AllMetrics.cs

```diff
@@ -0,0 +1,67 @@
+namespace NServiceBus.Metrics
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using global::Metrics;
+    using global::Metrics.Core;
+    using global::Metrics.Json;
+    using global::Metrics.MetricData;
+    using global::Metrics.Utils;
+
+    /// <summary>
+    /// Allows to define all avaible metrics
+    /// </summary>
+    public static class AllMetrics
+    {
+        internal static IMetricBuilder[] Create() => new IMetricBuilder[]
```

Comment:
@mikeminutillo I added a verification test. Now the reflection is in the test and not in the production code